### ISSUE TITLE
New plan for the personal page

### ISSUE THREAD

**Issue by rgoraya** [rgoraya/wc_master#83]
Documentation of the new plan for the personal page by @badgerProd 

Top left: Same as current.

Bottom: Replace with expanded version of what is currently at top right, with the following columns:
"Logo", "My Actions", "Community Acceptance", "Relationship Score", "Date"

Once that is in place, the top right should be replaced with an editable text-entry box for "Bio", editable email address, "Confirm email" button, and "Request Experience-Based Reputation Boost" button (ungrayed once email is confirmed).
